Rakefile

```diff
@@ -11,17 +11,25 @@ def brew_install(package, *args)
     # brew did not error out, verify tmux is greater than 1.8
     # e.g. brew_tmux_query = 'tmux 1.9a'
     installed_version = versions.split(/\n/).first.split(' ')[1]
-    unless version_match?(options[:version], installed_version)
+    unless version_match?(options[:requires], installed_version)
       sh "brew upgrade #{package} #{args.join ' '}"
     end
   end
 end
 
-def version_match?(requirement, version)
+def version_match?(requirement, v)
   # This is a hack, but it lets us avoid a gem dep for version checking.
+  version = parse_version(v)
```

Comment:
Strangest thing. I tried it in irb as well and it works as expected. But I also tried it again with a pry breakpoint just before this line. Same line `Gem::Dependency.new('', '>= 1.8').match?('', '1.9a')` returns false. Thoughts?
![screen shot 2015-05-07 at 3 33 27 pm](https://cloud.githubusercontent.com/assets/973173/7527299/b6630654-f4ce-11e4-9279-6049fc44c9dd.png)